config/settings.yml

```diff
@@ -1,5 +1,6 @@
 hold_button_url: http://cat.libraries.psu.edu/cgi-bin/catredirpg?C=
 matomo_id: 10
+report_issue_email_recipient: test@test.com
```

Comment:
@banukutlu - I talked to @awead about this and he mentioned preferring to store info like this in environment variables which could be pulled down from Vault. Is the catalog storing any data in environment variables right now, or are all the settings just in `settings.yml`?

Not sure if this is something that needs to be changed before this PR gets merged, but wanted to bring it up.

Reply:
We use Vault in Scholarsphere, which just sets environment variables. I'm _assuming_ there's something similar going on here. The gist was, don't put a real email in the repo here, and the deployment (QA, prod, etc.) would set this to the real email where you want things to go.

Reply:
We are using config gem to handle these, production.local.yml or settings.local.yml where we can add the real email.